sdl-core/src/main/scala/io/smartdatalake/app/StatusInfoRestApiConfig.scala

```diff
@@ -0,0 +1,11 @@
+package io.smartdatalake.app
```

Comment:
Copyright header is missing. Did you setup IntelliJ accordingly?
https://github.com/smart-data-lake/smart-data-lake/blob/develop-spark3/docs/Contribute.md#setup-license-header-in-intellij

Reply:
Sorry about that, I had disabled all plugins at some point and just realized that Copyright is also a plugin...